draft-ietf-quic-ack-frequency.md

```diff
@@ -494,9 +494,15 @@ estimate of the typical RTT, but not the minimum RTT (min_rtt)
 unnecessarily high number of acknowledgments when min_rtt is much smaller than
 smoothed_rtt.
 
-Note that the congestion window and the RTT change over the lifetime of a
+Note that the congestion window and the RTT estimate change over the lifetime of a
 connection and therefore might require sending frequent ACK_FREQUENCY frames to
-ensure optimal performance.
+ensure optimal performance, though not every change should trigger an update.
+Usually, it is not necessary to send an ACK_FREQUENCY frame more than once per
+RTT and likely it needs to be sent even less frequently.
+Ideally, an ACK_FREQUENCY frame is sent only if a relevant change
+in the congestion window or smoothed RTT is detected that impacts the local
+setting of the reordering threshold or locally-selected calculation of the
+Ack-Eliciting Threshold or Requested Max Ack Delay.
```

Comment:
I wonder if rearranging the paragraph would make things a bit more clear. Taking the proposed text and shuffling things a bit, something like

> Usually, it is sufficient to send an ACK_FREQUENCY frame once per RTT or at a
lower frequency. However, the congestion window and RTT estimate can change over
the lifetime of a connection, which might require sending frames at a higher
frequency in order to achive optimal performance. Not every change to congestion
window or RTT estimate warrants sending an ACK_FREQUENCY frame, ideally sending
occurs only when a change to congestion window or smoothed RTT impacts the local
setting of: the reordering threshold, or locally-selected calculation of the
Ack-Eliciting Threshold, or Requested Max Ack Delay.

Reply:
thanks for the proposal. I don't think your first sentence is putting the emphasis right. therefore I slightly prefer the current order of statements but I made some small wording updates that hopefully help. I will merge for now but if further editorial improvements are needed we can probably still address this later with another PR. Thanks!